src/csevrsn.in.h

```diff
@@ -19,6 +19,10 @@
 #define CSEVRSN_META "@CSEVRSN_META@"
 
 // version # as quoted text
+#ifdef CSE_OS_WINDOWS
 #define CSEVRSN_TEXT MAKE_LIT(CSEVRSN_MAJOR##.##CSEVRSN_MINOR##.##CSEVRSN_PATCH##@CSEVRSN_PRERELEASE@##) CSEVRSN_META
+#else
+#define CSEVRSN_TEXT MAKE_LIT(CSEVRSN_MAJOR.CSEVRSN_MINOR.CSEVRSN_PATCH@CSEVRSN_PRERELEASE@) CSEVRSN_META
+#endif
```

Comment:
1. I haven't looked into this yet, but is the C preprocessor syntax really different. I would be surprised if so. It would be good to add a comment to clarify why this distinction is necessary.
2. If it really is, is this an operating system difference or a compiler difference?

Reply:
I get several errors this is just one of them `pasting "CSEVRSN_MAJOR" and "." does not give a valid preprocessing token`. When I google this more recently I realized that the previous CSE version concatenation was wrong, and we should use the lower concatenation.